packages/cli/src/commands/index.ts

```diff
@@ -39,6 +39,14 @@ export function registerRepoCommand(program: Command) {
       '--all',
       'Build all packages, including bundled app and backend packages.',
     )
+    .option(
```

Comment:
Should I be adding a check for role = frontend as well? I think it would be confusing for the backend to allow both runtime config options and build time config (as well as build time config not being supported currently).

Reply:
Yep haven't quite figured out what makes sense here yet tbh. The fact that the backend needs the full frontend URL is what makes this complicated to begin with. Either way, the CLI options we provide here should not be forwarded to the backend, only the frontend. Don't think you can really do it any other way though, since the backend reads the config by itself.

Reply:
You know what? These options don't actually make sense for `repo build` now that I think about it. Let's only add them to the `package build` command for the frontend.